src/pages/fieldManagement/settings/defaultFieldComponents/SpeciesEditor.jsx

```diff
@@ -160,10 +184,20 @@ export default function SpeciesEditor({
             <FormattedMessage id="SEARCH" />
           </Button>
         </form>
+        {stillGeneratingDisplay && !searchResultsError && (
+          <Text
+            component="p"
+            variant="caption"
+            style={{ margin: '8px 4px' }}
+            id="STILL_GENERATING_LIST"
+          />
+        )}
         {searchResultsError && (
-          <CustomAlert severity="error">
-            {searchResultsError}
-          </CustomAlert>
+          <CustomAlert
+            style={{ marginTop: 12 }}
+            severity="error"
+            titleId="SEARCH_TIMED_OUT_WHILE_TRYING_TO_CONNECT_TO_ITIS"
+          />
```

Comment:
❗ We should still show the `searchResultsError` if it fails for some reason other than the timeout.

Reply:
Responded in https://github.com/WildMeOrg/codex-frontend/pull/471/commits/49db30f76452f80b3cdce12a2017a5600f193e96